plugins/inputs/kafka_consumer/sample.conf

```diff
@@ -6,6 +6,14 @@
   ## Topics to consume.
   topics = ["telegraf"]
 
+  ## Topic regular expressions to consume.  Matches will be added to topics.
+  # topic_regexps = [ "*test", "metric[0-9A-z]*" ]
```

Comment:
Commented out config options should express the default value, this seems not the case.

Reply:
The option itself is optional so there is no default in the traditional sense (except from being disabled/empty). But yeah if we are strict this should be an empty list.

Reply:
I'm rewriting the commentary to leave the defaults as the default but to put worked examples in the string.  While it's pretty straightforward for the refresh interval, I think we want an example of a list of regular expressions for matches.

Reply:
Fixed by @srebhan below.